request/tests/test-request.js

```diff
@@ -519,5 +519,19 @@ o.spec("xhr", function() {
 				o(e instanceof Error).equals(true)
 			}).then(done)
 		})
+		o("does not reject on error when extract provided", function(done) {
```

Comment:
Should we also add a test to ensure that a throwing `extract` still errors? Or is it already tested (in which this one needs its name clarified)?